src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -296,6 +328,8 @@ def load_arguments(self, _):
     with self.argument_context('backup restore restore-azurewl') as c:
         c.argument('vault_name', vault_name_type, id_part=None)
         c.argument('recovery_config', options_list=['--recovery-config'], help="""Specify the recovery configuration of a backed up item. The configuration object can be obtained from 'backup recoveryconfig show' command.""")
+        c.argument('rehydration_duration', type=int, help='Maximum time, in days, the recovery point stays in hydrated state.')
```

Comment:
Specify the default value which will be used in case user does not provide this input. 15 in your case.

Reply:
done